python/hiro-client.py

```diff
@@ -42,6 +43,8 @@ def __init__(self, username, password,
 
     @backoff.on_exception(*BACKOFF_ARGS, **BACKOFF_KWARGS)
     def get(self, url, token=None):
+        url = url.replace('"','') # temporary workaround.
```

Comment:
could you pls explain where " comes from?

Reply:
That's a bit weird, but for example in my IDE if i don't remove ", then it throws an exception from requests library.
At the same time I can successfully run this code directly on the server via console.
This was done mostly with a purpose of compatibility and I plan to improve it the future (re-write how this URL is forming: url = self._graph_endpoint + '/' + quote_plus(node_id))

<img width="1414" alt="Screenshot 2020-07-29 at 10 20 20" src="https://user-images.githubusercontent.com/36957194/88776714-9989f700-d186-11ea-86bf-acbf28b51c04.png">

Reply:
maybe you initialize it with quoted graph_endpoint instead of plain string url